lib/slack/real_time/client.rb

```diff
@@ -170,6 +174,7 @@ def restart_async
         @url = data.url
         @store = @store_class.new(data) if @store_class
         @socket.restart_async(self, @url)
+        @reconnected = true
```

Comment:
Assuming you need this (sure why not), since this can happen multiple times I would make this a timestamp, so `@reconnected_at = Time.now.utc`, then implement both `reconnected_at` and `reconnected?`. Either way separate PR please.